lms/templates/registration/password_reset_confirm.html

```diff
@@ -3,148 +3,70 @@
-        <span class="title-super">
-          {% blocktrans with platform_name=platform_name %}
-          Reset Your {{ platform_name }} Password
-          {% endblocktrans %}
-        </span>
-      </h1>
-    </header>
-  </section>
-
-  <section class="passwordreset container">
-    <section role="main" class="content">
-      {% if validlink %}
-      <header>
-      <h2 class="sr">{% trans "Password Reset Form" %}</h2>
-      </header>
-
-      <form role="form" id="passwordreset-form" method="post" action="">{% csrf_token %}
-        <!-- status messages -->
-        <div role="alert" class="status message">
-          <h3 class="message-title">
-            {% blocktrans %}
-                We're sorry, but this version of your browser is not supported. Try again using a different browser or a newer version of your browser.
-            {% endblocktrans %}
-          </h3>
-        </div>
-
-        {% if err_msg %}
-        <div role="alert" class="status message submission-error" style="display: block;">
-        {% else %}
-        <div role="alert" class="status message submission-error" style="display: none;">
-        {% endif %}
-          <h3 class="message-title">{% trans "The following errors occurred while processing your registration: " %}</h3>
-          <ul class="message-copy">
-            {% if err_msg %}
-              <li>{{err_msg}}</li>
+<div id="password-reset-confirm-container" class="login-register">
+    <section id="password-reset-confirm-anchor" class="form-type">
+        <div id="password-reset-confirm-form" class="form-wrapper">
+            <div class="status submission-error {% if not err_msg %} hidden {% endif %}" aria-live="polite">
+                <h4 class="message-title">{% trans "Error Resetting Password" %}</h4>
+                <ul class="message-copy">
+                {% if err_msg %}
+                    <li>{{err_msg}}</li>
+                {% else %}
+                    <li>{% trans "You must enter and confirm your new password." %}</li>
+                    <li>{% trans "The text in both password fields must match." %}</li>
+                {% endif %}
+                </ul>
+            </div>
+            {% if validlink %}
+
+            <form role="form" id="passwordreset-form" method="post" action="">{% csrf_token %}
+                <div class="section-title lines">
+                    <h2>
+                        <span class="text">
+                        {% trans "Reset Your Password" %}
+                        </span>
+                    </h2>
+                </div>
+
+                <p class="action-label">
+                    {% trans "Enter and confirm your new password." %}
+                </p>
+
+                <div class="form-field new_password1-new_password1">
+                    <label for="new_password1">{% trans "New Password" %}</label>
+                    <input id="new_password1" type="password" name="new_password1" placeholder="*****" />
```

Comment:
Can we get rid of this placeholder text?  It provides no value.  same for password_2 below.